spec/features/merchants/invoices/show_spec.rb

```diff
@@ -27,4 +29,25 @@
     expect(page).to have_content("Created on: #{@invoice1.created_at.strftime('%A, %B%e, %Y')}")
     expect(page).to have_content("Customer: #{@mary.first_name} #{@mary.last_name}")
   end
+
+  it 'display name/quantity/price/status for all invoice items' do
+    visit "/merchants/#{@merchant1.id}/invoices/#{@invoice1.id}"
+    
+    expect(page).to have_content(@item1.name)
+    expect(page).to have_content(@item2.name)
+    expect(page).to_not have_content(@item3.name)
+    expect(page).to have_content(@invoiceitem1.quantity)
+    expect(page).to have_content(@invoiceitem2.quantity)
+    expect(page).to have_content(@item1.unit_price)
+    expect(page).to have_content(@item2.unit_price)
+    expect(page).to have_content(@invoiceitem1.status)
+    expect(page).to have_content(@invoiceitem2.status)
+  end
+
+  it 'display the total revenue for items sold on this invoice' do
+    visit "/merchants/#{@merchant1.id}/invoices/#{@invoice1.id}"
```

Comment:
Can the "visit ____" be refactored to use advanced routing techniques? Can also be something to consider during refactoring after we get user stories done.